.gitignore

```diff
@@ -7,4 +7,6 @@
 *.iml
 *.ipr
 
-laundry-management-service/api/src/main/java/com/microservices/laundrymanagementapi/messages/
\ No newline at end of file
+#pattern for excluding generated proto files from git
+*Event.java
+LaundryState.java
```

Comment:
what about something like this: `**/*api/messages/` ?

Reply:
Do you want all the messages have same package? It's easier to distinguish which api we use we they have different packages and include service name

Reply:
But I agree We'd better to find some gentrified pattern for this

Reply:
i did as you said